src/comps/main/addComments/addComments.jsx

```diff
@@ -16,41 +19,72 @@ export const AddComments = props => {
 
   function handleSubmit(e) {
     e.preventDefault();
-    // console.log(comment.hasOwnProperty("comment"));
-    if (false) {
-      console.log("cant send, the comment is not valid");
+
+    /** Name: isEmpty()
+     * function checks if an object is empty
+     * @param {obj} obj
+     * return boolean
+     */
+
+    const isEmpty = obj => {
+      for (let key in obj) {
+        if (obj.hasOwnProperty(key)) return false;
+      }
+      return true;
+    };
+
+    if (isEmpty(comment)) {
+      alert("The comment is not valid!");
     } else {
       addCommentFunc(comment)
-        .then(res => alert("added new comment..."))
+        .then(res => {
+          alert("added new comment!");
+          loadComments();
```

Comment:
You already have the comment, why aren't you setting it to state instead of going to the server again to fetch the comment you just sent